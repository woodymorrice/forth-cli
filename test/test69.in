."pi" 3.14 variable pi . . ."pi" ."yum" set pi . .
