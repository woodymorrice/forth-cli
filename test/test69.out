3.1400
stack empty
yum
stack empty
